test/frontend.js

```diff
@@ -328,22 +328,22 @@ describe('Frontend', function () {
       const count = await n
         .click('.invite-button')
         .wait('#addButton')
-        .insert('#searchUser', username)
+        .insert('input[name="invitee"]', username)
         .click('#addButton')
-        .wait(() => document.querySelectorAll('.member').length > 0)
+        .wait(() => document.querySelectorAll('.invitee').length > 0)
```

Comment:
May I ask if we have any "strategy" to target elements used by integration tests?

By only seeing this PR, I can see we are using any kind of way to target an element (id, class, attr, etc..) without consistency. That might cause some confusion during code development. For instance, when I use classes, I expect to use them as a way to style the layout, not to be used as a test target. 

My suggestion here, would be to find a way to improve this lack of consistency. Example: Add an attr`data-test` to elements used to run tests. I see some advantages with that:
- During developing an element with that attribute is easily "marked" as an element used for tests.
- Any other element's property (class, id, attr) can be changed without worrying too much about breaking the test but at the same time keeping in mind that element is used for tests.
- These kind of changes (like this PR) can be avoid. 

I'm sorry if my question/suggestion is redundant and this topic was already discussed, I didn't find anything about it.

Reply:
> For instance, when I use classes, I expect to use them as a way to style the layout, not to be used as a test target.

That makes sense, yes. Unless for some reason the test needs to verify there are some elements that have that class, which could happen.

> My suggestion here, would be to find a way to improve this lack of consistency. Example: Add an attr `data-test` to elements used to run tests.

Yes, that sounds like a great plan! 👍  Feel free to do that, and along the way check with @hubudibu about cleaning up and removing an no-longer-used IDs or classes.

Reply:
sounds great, nice and clean, it would make testing much easier 👍